inspec.yml

```diff
@@ -0,0 +1,377 @@
+    <% end %>
+
+  - name: pass_lock_time # V-73313
+    desc: "List the number of minutes before a session is locked out"
+    type: Numeric
+    value: 120
+    profile: microsoft-windows-server-2016-stig-baseline
+
+  - name: pass_lock_duration # V-73309
+    desc: "List the number of minutes for account lockout duration"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'High'].include? ENV['BASELINE'] %>
+    value: 0
+    <% elsif ['Moderate-HVA', 'Moderate'].include? ENV['BASELINE'] %>
+    value: 60
+    <% elsif ['Low-HVA', 'Low'].include? ENV['BASELINE'] %>
+    value: 15
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 60
+    <% end %>
+
+  - name: app_password_age # V-73231
+    desc: "Set the number of days that manually managed application account passwords must be changed"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 60
+    <% end %>
+ 
+  - name: maximum_password_age_machine # V-73641
+    desc: "Set the machine account maximum password age stated by your organiztion"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 60
+    <% end %>
+
+  - name: unused_account_age # V-73259
+    desc: "Set the maximum age in days for unused accounts stated by your organization"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'High'].include? ENV['BASELINE'] %>
+    value: 30
+    <% elsif ['Moderate-HVA', 'Moderate'].include? ENV['BASELINE'] %>
+    value: 60
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 60
+    <% end %>
+
+  - name: minimum_password_length_manual # V-73229
+    desc: "Set the minimum password length for manually managed applications stated by your organization"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 20
+    <% elsif ['High', 'Moderate', 'Low'].include? ENV['BASELINE'] %>
+    value: 15 # Same as the baseline parameter
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 15
+    <% end %>
+
+  - name: minimum_password_length # V-73321
+    desc: "Set the minimum password length stated by your organization"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 20
+    <% elsif ['High', 'Moderate', 'Low'].include? ENV['BASELINE'] %>
+    value: 15
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 15
+    <% end %>  
+
+  # V-73319 minimum_password_age input value is the same as the baseline parameter of 1
+  # V-73317 maximum_password_age input value is the same as the baseline parameter of 60
+  
+  - name: password_history_size # V-73315
```

Comment:
This input is currently not used so it's leading to an accurate warning

Reply:
is there any action needed from me?